src/main/java/net/minecraftforge/event/entity/EntityTeleportEvent.java

```diff
@@ -88,7 +89,7 @@ public TeleportCommand(Entity entity, double targetX, double targetY, double tar
 
     /**
      * EntityTeleportEvent.SpreadPlayersCommand is fired before a living entity is teleported
-     * from use of {@link net.minecraft.command.impl.SpreadPlayersCommand}.
+     * from use of {@link net.minecraft.server.commands.SpreadPlayersCommand}.
```

Comment:
Import inconsistency.

Reply:
This cannot be changed to an import, as it would then conflict with the `SpreadPlayersCommand` inner class.